decidim-core/app/views/layouts/decidim/footer/_main.html.erb

```diff
@@ -6,7 +6,7 @@
     <div class="lg:w-2/3 grid grid-cols-1 gap-x-4 gap-y-10 lg:grid-cols-4 text-md text-white">
       <%= render partial: "layouts/decidim/footer/main_links" %>
     </div>
-    <%= cell("decidim/footer_pages", :topics) %>
+    <%= cell("decidim/footer_topics", :topics) %>
```

Comment:
As the parameter isn't used, it should be nil (as I mentioned in my other comment)

```suggestion
    <%= cell("decidim/footer_topics", nil) %>
```